### DISCUSSION TITLE
去日本旅遊

### DISCUSSION THREAD

**Discussion by seastar0687** [1101-GS4719-C/Python-Community-GS4719-C#3706] [General]
寒假快到了，大家會想出國旅遊嗎
我自己已經規劃好了要去日本的京都、大阪跟神戶旅遊了
並且規劃在最有機會降學的2月初左右
我覺得在學生時期多出去旅遊是一個很好事情，無論是增加國際視野或是對於第二外語的培訓都很有幫助
甚至是獨自出國的話還能培養自立能力
所以我在今年也成功去了兩次日本東京，一次是獨自出國
希望大家也都能出國去玩

**Comment by poyen0806** [comment]
我是很想出國旅遊，體驗看看不同文化、風景
但是看到最近飛機失事的新聞後
就覺得要先把出國計畫的想法收起來了
下次一定


可以用request函式庫跟beautifulsoup或selenium等
利用爬蟲蒐集一下旅遊資訊
還有API可以查詢天氣
